add class procedure

pablopers opened the pull request:
add class procedure

pablopers committed:
Update requirements.txt

update new theme

pablopers committed:
Update conf.py

edit for new theme

pablopers committed:
Update index.rst

pablopers committed:
Update index.rst

pablopers committed:
Update conf.py

pablopers committed:
Patch 8 (#7)

* Update scrivere-un-documento.rst

Aggiunta del tipo di file ODT per il convertitore web

* Aggiunta virgola

* Aggiunti campi obbligatori

* Aggiunti campi obbligatori nella tabella

* Segnalati campi obbligatori yml

* Aggiunto document_settings

* Aggiunti numeri sezione

* Test link figure

* Update appendice-3.rst

edit image in the table (test)

* Update appendice-3.rst

edit procedure

* Update appendice-3.rst

pablopers committed:
Patch 7 (#6)

* Update scrivere-un-documento.rst

Aggiunta del tipo di file ODT per il convertitore web

* Aggiunta virgola

* Aggiunti campi obbligatori

* Aggiunti campi obbligatori nella tabella

* Segnalati campi obbligatori yml

* Aggiunto document_settings

* Aggiunti numeri sezione

* Test link figure

* Update appendice-2.rst

edit properties image

* Update appendice-2.rst

* Update appendice-2.rst

pablopers committed:
Patch 6 (#5)

* Update scrivere-un-documento.rst

Aggiunta del tipo di file ODT per il convertitore web

* Aggiunta virgola

* Aggiunti campi obbligatori

* Aggiunti campi obbligatori nella tabella

* Segnalati campi obbligatori yml

* Aggiunto document_settings

* Aggiunti numeri sezione

* Test link figure

* Update appendice-1.rst

edit properties image

pablopers committed:
Patch 5 (#4)

* Update scrivere-un-documento.rst

Aggiunta del tipo di file ODT per il convertitore web

* Aggiunta virgola

* Aggiunti campi obbligatori

* Aggiunti campi obbligatori nella tabella

* Segnalati campi obbligatori yml

* Aggiunto document_settings

* Aggiunti numeri sezione

* Test link figure

* Update gestione-dei-documenti.rst

edit properties image

pablopers committed:
Patch 4 (#3)

* Update scrivere-un-documento.rst

Aggiunta del tipo di file ODT per il convertitore web

* Aggiunta virgola

* Aggiunti campi obbligatori

* Aggiunti campi obbligatori nella tabella

* Segnalati campi obbligatori yml

* Aggiunto document_settings

* Aggiunti numeri sezione

* Test link figure

* Update pubblicare-un-documento.rst

edit properties image

pablopers committed:
Patch 3 (#2)

* Update scrivere-un-documento.rst

Aggiunta del tipo di file ODT per il convertitore web

* Aggiunta virgola

* Aggiunti campi obbligatori

* Aggiunti campi obbligatori nella tabella

* Segnalati campi obbligatori yml

* Aggiunto document_settings

* Aggiunti numeri sezione

* Test link figure

* Update come-partecipare.rst

edit properties image

pablopers committed:
Patch 2 (#1)

* Update scrivere-un-documento.rst

Aggiunta del tipo di file ODT per il convertitore web

* Aggiunta virgola

* Aggiunti campi obbligatori

* Aggiunti campi obbligatori nella tabella

* Segnalati campi obbligatori yml

* Aggiunto document_settings

* Aggiunti numeri sezione

* Test link figure

* Update che-cos-e-docs-italia.rst

edit proprieties image

* Update che-cos-e-docs-italia.rst

pablopers committed:
Update pubblicare-un-documento.rst

pablopers committed:
Update che-cos-e-docs-italia.rst

pablopers committed:
Update appendice-3.rst

pablopers committed:
Update come-partecipare.rst

pablopers committed:
Update come-partecipare.rst

pablopers committed:
Update come-partecipare.rst

pablopers committed:
Update come-partecipare.rst

pablopers committed:
Update come-partecipare.rst

pablopers committed:
Update appendice-2.rst

pablopers committed:
Update pubblicare-un-documento.rst

new class procedure

pablopers committed:
Update scrivere-un-documento.rst (#10)

add two class procedure

alterat commented on index/pubblicare-un-documento.rst:
```diff
@@ -529,17 +529,17 @@ Per creare un documento multilingue è necessario creare i documenti nelle diver
-| -  Clicca sul pulsante **Submit**.                                                                                                     |
-+----------------------------------------------------------------------------------------------------------------------------------------+
+
+.. topic:: Procedura. Impostare la traduzione di un documento
```
Credo che qui sia necessario lasciare `.. topic:: Procedura` e mettere il titolo come `internal-title`. Così:

```
.. topic:: Procedura
   :class: procedure
   :procedure-internal-title: Impostare la traduzione di un documento
```

pablopers committed:
Update requirements.txt

pablopers committed:
Update conf.py

pablopers committed:
Update conf.py

pablopers committed:
Update conf.py

pablopers committed:
Update conf.py

pablopers committed:
Update conf.py

pablopers committed:
Update conf.py

pablopers committed:
Merge branch 'master' into patch-2

pablopers committed:
Patch 2 (#12)

* Update scrivere-un-documento.rst

Aggiunta del tipo di file ODT per il convertitore web

* Aggiunta virgola

* Aggiunti campi obbligatori

* Aggiunti campi obbligatori nella tabella

* Segnalati campi obbligatori yml

* Aggiunto document_settings

* Aggiunti numeri sezione

* Test link figure

* Update che-cos-e-docs-italia.rst

edit proprieties image

* Update che-cos-e-docs-italia.rst

* Update come-partecipare.rst

edit properties image

* Update pubblicare-un-documento.rst

edit properties image

* Update gestione-dei-documenti.rst

edit properties image

* Update appendice-1.rst

edit properties image

* Update appendice-2.rst

edit properties image

* Update appendice-2.rst

* Update appendice-2.rst

* Update che-cos-e-docs-italia.rst

* Update pubblicare-un-documento.rst

* Modifiche cosmetiche

* Modifiche cosmetiche

* Modifiche cosmetiche

* Rimosso titolo doppio

* Update pubblicare-un-documento.rst

new class procedure

* Update scrivere-un-documento.rst (#10)

add two class procedure

* Update requirements.txt

* Update conf.py

* Update conf.py

* Update conf.py

* Update conf.py

* Update conf.py

* Update conf.py

pablopers committed:
Patch 1 (#11)

* Update scrivere-un-documento.rst

Aggiunta del tipo di file ODT per il convertitore web

* Aggiunta virgola

* Aggiunti campi obbligatori

* Aggiunti campi obbligatori nella tabella

* Segnalati campi obbligatori yml

* Aggiunto document_settings

* Aggiunti numeri sezione

* Test link figure

* Update che-cos-e-docs-italia.rst

edit proprieties image

* Update che-cos-e-docs-italia.rst

* Update come-partecipare.rst

edit properties image

* Update pubblicare-un-documento.rst

edit properties image

* Update gestione-dei-documenti.rst

edit properties image

* Update appendice-1.rst

edit properties image

* Update appendice-2.rst

edit properties image

* Update appendice-2.rst

* Update appendice-2.rst

* Update che-cos-e-docs-italia.rst

* Update pubblicare-un-documento.rst

* Modifiche cosmetiche

* Modifiche cosmetiche

* Modifiche cosmetiche

* Rimosso titolo doppio

* Update come-partecipare.rst

corretto sec rotto

* Update pubblicare-un-documento.rst (#8)

corretti sec rotti

pablopers committed:
class procedure (#9)

* Update appendice-3.rst

edit image in the table (test)

* Update appendice-3.rst

edit procedure

* Update appendice-3.rst

* Update appendice-3.rst

pablopers committed:
Merge branch 'master' into patch-2

pablopers committed:
Delete conf.py

pablopers committed:
Delete index.rst

pablopers committed:
Delete requirements.txt

pablopers commented:
Chiudo questa PR in favore di una nuova dove saranno riportate correttamente le procedure così come concordato con @atorin.
